### PR TITLE
EPMKHRDMH-348 [settings] [mobile] Mobile user settins

### PR DESCRIPTION
**Pull request by 8bitmousepunk**

### CODE REVIEW

File: mobile/modules/users/config/users.client.routes.es6 [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
+    }
+  }).state('restricted.booked', {
+    url: '/settings/booked',
+    views: {
+      'main-view': {
+        templateUrl: 'modules/users/views/settings/booked-settings.client.view.html',
+        controller: 'SettingsController',
+        controllerAs: 'settings'
+      }
+    }
```
Comment (ghost) [inline mobile/modules/users/config/users.client.routes.es6]: we don't need this state and view for it as we don't implement this logic

File: mobile/modules/users/less/_users.settings.less [hunk 2]
```diff
@@ -0,0 +1,190 @@
+@import (once) "../../../less/_app.variables.less";
+@import (once) "../../../less/_app.mixins.less";
+
+.font-size(@sizeValue) {
+  @remValue: @sizeValue / 10;
+  font-size: ~"@{remValue}rem";
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: please move this logic to 'less' variables

File: mobile/modules/users/less/_users.settings.less [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+}
+
+.right-item {
+  float: right;
+  margin-left: 20px;
+}
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: seems that it is a common rule for whole app (not only for settings)

File: mobile/modules/users/less/_users.settings.less [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+.settings {
+  background: @color-sidebar-separator;
+
+  height: 100%;
+
+  .main-info {
+    background: @color-blue-whale-blue-s53;
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: could you use component variable (mapped from color)

File: mobile/modules/users/less/_users.settings.less [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+  .main-info--username {
+    .font-size(16);
+  }
+
+  .main-info--email {
+    .font-size(14);
+  }
+
+  .main-info--upload {
+    color: @color-bali-hai-blue-s24;
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: could you use component variable (mapped from color)

File: mobile/modules/users/less/_users.settings.less [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+  }
+
+  .avatar {
+    width: 95px;
+    height: 95px;
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: could we use variable for this?

File: mobile/modules/users/less/_users.settings.less [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+    background: transparent;
+
+    color: @color-white;
+    .font-size(16);
+
+    outline: none;
+
+    option {
+      color: #000000;
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: could you move the color to variables?

File: mobile/modules/users/less/_users.settings.less [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+  .settings-header {
+    flex-grow: 0;
+  }
+
+  .settings-content {
+    flex-grow: 0;
+  }
+}
+
+.settings-header {
```
Comment (ghost) [inline mobile/modules/users/less/_users.settings.less]: please use common styles for header

**Review by ghost — CHANGES_REQUESTED** [review]

File: mobile/modules/users/less/_users.settings.less [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+    box-sizing: content-box;
+
+    padding: 10px 0;
+
+    border: 0 none;
+    border-bottom: 1px solid @settings-sub-font-color;
+
+    background: transparent;
+
+    color: @color-white;
```
Comment (make-it-happen-team) [inline mobile/modules/users/less/_users.settings.less]: could you use component variable (mapped from color)

File: mobile/modules/users/less/_users.settings.less [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+
+  .main-info--email {
+    .font-size(14);
+  }
+
+  .main-info--upload {
+    color: @settings-sub-font-color;
+  }
+
+  @avatar-size: 95px;
```
Comment (make-it-happen-team) [inline mobile/modules/users/less/_users.settings.less]: please declare variables on the top of the file

**Review by make-it-happen-team — CHANGES_REQUESTED** [review]

**Pull request merged**